docs/l2-l3.rst

```diff
@@ -212,11 +216,9 @@ Create Self IPs
 ```````````````````````````
 
 1. Click the L2-L3 Network Management tab.
+2. Select one or more available vlans and move them to the Selected box. These VLANs are already present on the vCMP Guest BIG-IP device After selecting the VLANS click Manage Selected.
```

Comment:
..available VLANs
<seems like you are missing a period after Guest BIG-IP device.. either that or comma and no cap 'after'>